zio-2/src/main/scala/com/baeldung/scala/zio/logging/BaeldungLogger.scala

```diff
@@ -0,0 +1,49 @@
+package com.baeldung.scala.zio.logging
+
+import zio.*
+import zio.logging.backend.SLF4J
+import zio.logging.{
+  LogColor,
+  LogFilter,
+  LogFormat,
+  LoggerNameExtractor,
+  console,
+  fileJson
+}
+import zio.logging.LogFormat.*
+
+import java.nio.file.Path
+import java.time.format.DateTimeFormatter
+
+object BaeldungLogger {
+
+  private val filter =
+    LogFilter.logLevelByName(
+      LogLevel.Trace,
```

Comment:
Did you mean to use Trace and Info on this line and the next line - per config, only logging Info?

Reply:
Got it - let's add a line indicating this dependency is added during the article